src/main/java/com/starbucks/service/OrderService.java

```diff
@@ -18,39 +17,31 @@ public OrderService(OrderRepository orderRepository) {
     }
 
     public String saveOrder(Order order) {
-
-        orderRepository.save ( order );
-        return order.toString ( );
+        orderRepository.save(order);
+        return order.toString();
     }
 
-    public String updateOrder(Order order , UUID id) {
-
-        if (orderRepository.existsById ( id )) {
-            Order orderDB = orderRepository.findById ( id ).get ( );
-            orderDB.setBeverages ( order.getBeverages ( ) );
-            orderDB.setSize ( order.getSize ( ) );
-            orderDB.setSugar ( order.getSugar ( ) );
-            orderRepository.save ( orderDB );
-        }
-        return "Order " + id + " not exist";
-
+    public String updateOrder(UUID id, Order order) {
+        Order orderDB = orderRepository.findById(id).get();
+        orderDB.setBeverages(order.getBeverages());
+        orderDB.setSize(order.getSize());
+        orderDB.setSugar(order.getSugar());
+        orderRepository.save(orderDB);
+        return "Order updated!";
```

Comment:
Awesome :) , but you should return the updated object, just put return before orderRepository.save(orderD B);